scripts/test/add_type_to_factory.cdc

```diff
@@ -0,0 +1,16 @@
+import "CapabilityFactory"
+import "NFTProviderFactory"
+
+import "NonFungibleToken"
+
+pub fun main(address: Address, type: Type): Bool {
+    let managerRef = getAuthAccount(address).borrow<&CapabilityFactory.Manager>(
+        from: CapabilityFactory.StoragePath
+    ) ?? panic("CapabilityFactory Manager not found")
+    
+    let nftProviderFactory = NFTProviderFactory.Factory()
+    
+    managerRef.addFactory(type, nftProviderFactory)
```

Comment:
Similar comment as made for `add_type_to_factory.cdc` - recommend acknowledging NFTProviderFactory in the filename